<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рекомендательная система музыкальных исполнителей</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="rec_widget_">
     <widget class="QWidget" name="rec_id_tab_">
      <attribute name="title">
       <string>По предпочтениям пользователя</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="id_h_layout_">
         <item>
          <widget class="QLabel" name="id_label_">
           <property name="text">
            <string>ID пользователя:</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="id_spin_box_"/>
         </item>
         <item>
          <spacer name="id_h_spacer_">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="id_artist_h_layout_">
         <item>
          <layout class="QVBoxLayout" name="preffered_v_layout_">
           <item>
            <widget class="QLabel" name="preffered_label_">
             <property name="text">
              <string>Предпочитаемые исполнители</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="preffered_list_widget_"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rec_button_v_layout_">
           <item>
            <spacer name="rec_button_v_spacer_">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="rec_push_button_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Рекомендовать</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="rec_button_v_spacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rec_artist_v_layout_">
           <item>
            <widget class="QLabel" name="rec_artist_label_">
             <property name="text">
              <string>Рекомендуемые исполнители</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="rec_artist_list_widget_"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rec_artist_tab_">
      <attribute name="title">
       <string>По исполнителю</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="artist_h_layout_">
         <item>
          <widget class="QLabel" name="artist_label_">
           <property name="text">
            <string>Исполнитель:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="artist_line_edit_"/>
         </item>
         <item>
          <spacer name="artist_h_spacer_">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="search_push_button">
           <property name="text">
            <string>Найти похожих исполнителей</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="artist_list_widget_"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
